docs/run_test/run_locally.rst

```diff
@@ -0,0 +1,33 @@
+Running Tests Locally
```

Comment:
1. The title should be like "Run tests in Local VM". It's more accurate. The LISA can run in local without a VM.
2. Link it to some page, creating an orphan page is not a good practice.

Reply:
My bad! I thought the index was automatically created from directory structure by sphinx.